spec/lib/rmt/cli/products_spec.rb

```diff
@@ -198,6 +198,22 @@
           expect(repository.mirroring_enabled).to eq(false)
         end
       end
+
+      context 'with recommended extensions' do
+        let(:product) { create :product, :with_mirrored_repositories, :with_recommended_extensions }
+        let(:extensions) { Product.recommended_extensions(product).to_a }
+
+        it 'disabled the mandatory product repositories' do
```

Comment:
I think it would disable all of the product repos (not just the mandatory ones). I see that the example that we've previously had has the same text, but it's misleading.